dauphine-open-data-lib/src/main/java/io/github/oliviercailloux/y2018/opendata/entity/Person.java

```diff
@@ -0,0 +1,264 @@
+package io.github.oliviercailloux.y2018.opendata.entity;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import com.google.common.base.Strings;
+
+/**
+ * Provides a basic implementation of class Person.<br />
+ * Person is either Student or Teacher. In this class you have basic attributes to
+ * manipulate class Person and getters an setters
+ * <br />
+ * @author Gandi Taric
+ */
+
+@Entity
```

Comment:
C’est une entité, mais il n’y a pas d’annotation sur les listes, ça ne va pas fonctionner. (Cf. prochaine séance.)

Reply:
Ok bien noté je corrigerai cela

Reply:
> Ok bien noté je corrigerai cela

Comment ça, « corrigerai » ? J’attends votre correction pour pouvoir approuver ce PR.
Notez, pour éviter toute confusion, que la « prochaine séance » à laquelle mon commentaire du 12 janvier faisait référence était la S5 du 14 janvier. J’imagine que c’était clair pour vous.
Quoi qu’il en soit, le problème existe toujours. Avez-vous testé votre code ? Ça m’étonnerait qu’il fonctionne.

Reply:
@oliviercailloux 
Suite à notre discussion d'aujourd'hui et vu que mes listes sont sur des string, ma façon de déclarer est correct.
Je vous prie d'en tenir compte pour le retard  de ma soumission car j'ai beaucoup cherché à comprendre mon erreur , en demandant à mes camarades aussi.
Je vous remercie.

Reply:
@oliviercailloux  
Bonjour Monsieur c'est maintenant Ok?